namespaces/Representor.md

```diff
@@ -0,0 +1,44 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - Any embedded (resources) from this resource.
```

Comment:
> they should be in the content

I would say embedded resources can be omitted when serializing a resource. The actual content of a resource is NOT any embedded resource but the resource data itself. To me it make much sense to have embedded resources in attributes. 

Maybe the whole problem of why this doesn't feel right is the use of word "attributes" together with mixing "plain hash of plain key: values" vs. "everything is element"